### DISCUSSION TITLE
I have a feeling that the english translation is not accurate

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#108] [Q&A]
https://github.com/rdrg109/subtitles-npcr/blob/f486e762694e210282ed573692532503e9820913/sentences/44-1.yaml#L191-L195

**Comment by rodrigo-morales-1** [comment]
I'm converting this issue into a discussion, because there is no request to modify any of the files in the repository, but instead this is a question.

**Comment by fhxshrs** [comment]
"rest assured" 后面跟的是不是都是句子？我也觉得这句翻译不太准确，你有更好的建议吗？

**Reply by rodrigo-morales-1** [reply]
@fhxshrs I agree with you, we can't add a sentence after "rest assured". I searched some examples on the Internet, but didn't find any. Searching `site:theguardian.com "rest assured to"` on Google showed 0 results and `site:bbc.com "rest assured to"` also showed 0 results.

The translation of 放心 in [https://trainchinese.com](https://www.trainchinese.com/v2/wordDetails.php?rAp=0&wordId=130025&tcLanguage=en) is "Rest assured". The definition of "rest assured" in [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/rest-assured) is:

> If you tell someone that they can rest assured about something, you mean that they can feel confident that it is true or that it will happen: 

DeepL shows the translation "You can buy with confidence". It uses the word "confidence" which is mentioned by Cambridge Dictionary. I think it makes sense to use: "You can buy with confidence." Do you agree with me?

**Reply by fhxshrs** [reply]
同意
